fmatch/matcher.py

```diff
@@ -125,13 +132,16 @@ def get_uuid_by_metadata(
         )
         result = self.query_index(index, s)
         hits = result.hits.hits
-        uuids_docs = [
-            {
-                "uuid": hit.to_dict()["_source"]["uuid"],
-                "buildUrl": hit.to_dict()["_source"]["buildUrl"],
-            }
-            for hit in hits
-        ]
+        uuids_docs = []
+        for hit in hits :
+            if "buildUrl" in hit["_source"] :
+                uuids_docs.append({
+                        "uuid": hit.to_dict()["_source"]["uuid"],
+                        "buildUrl": hit.to_dict()["_source"]["buildUrl"]})
+            else :
+                uuids_docs.append({
+                        "uuid": hit.to_dict()["_source"]["uuid"],
+                        "buildUrl": "http://bogus-url"})
```

Comment:
I am okay with rest of the code changes, but here for the `buildUrl` do you plan to parameterize this? If yes, I believe this hardcoded value is just a placeholder for now.

Reply:
Not following

Reply:
I was confirming your logic through a question. But I think I understood the intent. looks good.